Exiled.Events/Patches/Events/Map/ChangingIntoGrenade.cs

```diff
@@ -90,8 +90,8 @@ private static IEnumerable<CodeInstruction> Transpiler(IEnumerable<CodeInstructi
                 new CodeInstruction(OpCodes.Callvirt, PropertySetter(typeof(Grenades.Grenade), nameof(Grenades.Grenade.NetworkfuseTime))),
             });
 
-            foreach (CodeInstruction instruction in newInstructions)
-                yield return instruction;
+            for (int i = 0; i < newInstructions.Count; i++)
```

Comment:
I know that `i` is fine, but we used `z` for every other events.

Reply:
fixed